CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 ### Added
 - `vcs.remote_name` that tracks the name of the remote repository (e.g.: `default` on `hg` or `origin` on `git`)
 - `base_ref` parameter that points to the reference (e.g.: a branch) on the base repository
+- `vcs.find_latest_common_revision()` which find the common ancestor between a provided `base_ref` and the current `head_rev`
```

Comment:
nit: finds

Reply:
Looks like you may have lost the fix somewhere :)

Reply:
I think there was some confusion because I didn't click on the "Resolve conversation" button. I confirm I made the fix which is visible in this comment. Closing conversation to prevent further confusion 🙂 

[1] https://github.com/taskcluster/taskgraph/pull/93/commits/d46b95744be12516be4c15678083cb23994f99d8#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR15